### PR TITLE
Use specific rockylinux version in the workflow [5.0.z]

### PR DESCRIPTION
**Pull request by ldziedziul**
latest tag is not available anymore

Backport of https://github.com/hazelcast/hazelcast-packaging/pull/147

### CODE REVIEW

File: .github/workflows/publish-packages.yml [hunk 1]
```diff
@@ -183,7 +183,7 @@ jobs:
 
       - name: Install Required tools
         run: |
-          yum install -y maven rpm-sign rpm-build wget
+          yum install -y maven rpm-sign rpm-build wget gettext
```
Comment (TomaszGaweda) [inline .github/workflows/publish-packages.yml:186]: Is this related or by mistake?

**Review by TomaszGaweda — APPROVED** [review]

File: .github/workflows/publish-packages.yml (hunk 1 above)
Comment (ldziedziul) [reply .github/workflows/publish-packages.yml:186]: it's related: after the upgrade `envsubst` command is not available. It's part of the `gettext` package
